src/main/src/main/java/org/geoserver/catalog/impl/FeatureTypeInfoImpl.java

```diff
@@ -34,6 +34,8 @@
 
     protected int maxFeatures;
     protected int numDecimals;
+    protected boolean padWithZeros;
+    protected boolean forcedDecimal;
```

Comment:
I just noticed this now, but you should probably be including these two new parameters in the equals() and hashCode() implementations